backend/src/test/java/com/woowacourse/levellog/presentation/FeedbackControllerTest.java

```diff
@@ -366,4 +367,88 @@ private ResultActions requestFindAllFeedback(final Long levellogId) throws Excep
             return requestGet("/api/levellogs/" + levellogId + "/feedbacks");
         }
     }
+
+    @Nested
+    @DisplayName("findById 메서드는")
+    class FindById {
+
+        @Test
+        @DisplayName("존재하지 않는 레벨로그 정보로 피드백 목록 조회를 요청하면 예외가 발생한다.")
+        void findById_notFoundLevellog_exceptionThrown() throws Exception {
```

Comment:
```suggestion
        void findById_notFoundLevellog_exception() throws Exception {
```
https://github.com/woowacourse-teams/2022-levellog/issues/124
컨벤션 확인해주세요 😉